package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@cortexapps/backstage-plugin",
-  "version": "1.5.0",
+  "version": "1.6.0",
```

Since this is purely a bug fix, we should do a patch version bump for this.
```suggestion
  "version": "1.5.1",
```